PERFORMANCE.md

```diff
@@ -0,0 +1,127 @@
+the first iteration time.
+
+## Inference Standard
+
+The following are scaling results from the cpp-inference scaling tests with ResNet-50
+and the imagenet dataset. For more information on these scaling tests, please see
+the SmartSim paper on arXiv
+
+![Inference plots dark theme](/figures/all_in_one_violin_dark.png#gh-dark-mode-only "Standard inference")
+![Inference plots ligh theme](/figures/all_in_one_violin_light.png#gh-light-mode-only "Standard inference")
+
+## Colocated Inference
+
+The following are scaling results for a colocated inference test, run on 12 36-core Intel Broadwell nodes,
+each one equipped with 8 Nvidia V100 GPUs. On each node, 28 client threads were run, and the databases
+were run on 8 CPUs and 8 threads per queue. 
+
+![Colocated inference plots dark theme](/figures/colo_dark.png "Colocated inference")
+![Inference plots ligh theme](/figures/colo_light.png "Colocated inference")
+
+## Inference Performance Analysis
+
+INSERT ANALYSIS - am rerunning the inference tests : ignore Inference section for this review
+
+## Throughput Standard
+The following are scaling results for a standard throughput test, run with 48 client threads on each node. The databases were run on 8 CPUs and 8 threads per queue. 
+
+#### Unpack Tensor - retrieve the data
+![Throughput Std Unpack](/figures/unpack_tensor_thro_std.png "Throughput Standard")
+
+#### Put Tensor - send image to database
+![Throughput Std Put](/figures/put_tensor_thro_std.png "Throughput Standard")
+
+## Throughput Colocated
+The following are scaling results for a colocated throughput test, run with 48 client threads on each node. The databases were run on 8 CPUs and 8 threads per queue. 
+
+#### Unpack Tensor - retrieve the data
+![Throughput colo Unpack](/figures/unpack_tensor_thro_colo.png "Colocated Throughput")
+
+#### Put Tensor - send image to database
+![Throughput colo put](/figures/put_tensor_thro_colo.png "Colocated Throughput")
+
+## Throughput Performance Analysis
+
+1. The Throughput Standard test density decreases as the database nodes increase. Outliers become greater as client total increases. This is because as there are more clients, there is a longer wait time. We can also tell that the standard takes longer than colocated because the client nodes are having to travel offsight to the database client node. Each database is having to handle more requests since they do not 
+have their own clients. 
+
+2. Put/Unpack tensor for Throughput Colocated stays consistent with density across all client totals.
+It takes much less time to complete using a colocated orchestrator since all the computations are happening on the same node. Less time it takes to communicate since the db and app are on the same node.
+
+## Data Aggregation Standard
+
+The following are scaling results for a data aggregation test, run with 32 client threads on each node. The databases were run on 36 CPUs and 4 threads per dataset. 
```

Comment:
Not sure what you mean by 4 threads per dataset? Also I think you might be mixing up threads and client count. It's 32 clients per node.

Also, you need to explain what the purpose of this test is and its functionality.